Dangerfile

```diff
@@ -4,10 +4,18 @@ gemfile_updated = !git.modified_files.grep(/Gemfile/).empty?
 warn 'The `Gemfile` was updated' if gemfile_updated
 
 # Rubocop lint
-rubocop.lint
+rubocop.lint(
+  force_exclusion: true,
+  inline_comment: true,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/HamstringAssassin/danger-spelling/blob/6b6d9ac83ad76a9a29621e57611312621e23c870/Dangerfile#L9"></span>Avoid comma after the last parameter of a method call.</td>
    </tr>
  </tbody>
</table>